index.html

```diff
@@ -457,5 +457,69 @@ <h3>
 
 </div>
 
+
+<div id="heading-grid">
+    <h4>
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </h4>
+</div>
+
+<form id="form-grid" action="https://formspree.io/f/xvodvrjv"
+method="POST">
+    <input type="text" placeholder="Full name" name="full-name" maxlength="30" required>
+    <input type="email" placeholder="Email address" name="email" required>
+    <textarea placeholder="Enter text here" name="comment-section" maxlength="500" required>Enter text here</textarea>
+    <div class="B-2">
+        <button type="submit" name="comment-section">
+           Get in touch
+        </button> 
```

Comment:
- [ ] Add the behavior design according to the Figma design you selected. 
- [ ] [OPTIONAL] Add the next CSS style `cursor: pointer;` for your buttons (when you hover them) to improve the UX.

![image](https://user-images.githubusercontent.com/60164239/129866180-89e03088-0511-4a8a-9f5c-5ba7b47549ff.png)